src/main/java/ooo/oshi/foreign/windows/WtsApi32.java

```diff
@@ -0,0 +1,63 @@
+package ooo.oshi.foreign.windows;
+
+import java.lang.foreign.*;
+import java.lang.invoke.MethodHandle;
+
+import static java.lang.foreign.ValueLayout.*;
+
+public class WtsApi32 {
+
+    private static final SymbolLookup WTSAPI32;
+    private static final Linker LINKER;
+
+    static {
+        LINKER = Linker.nativeLinker();
+        System.loadLibrary("wtsapi32");
+        WTSAPI32 = SymbolLookup.loaderLookup();
+    }
+
+    private static MethodHandle methodHandle(String methodName, FunctionDescriptor fd) {
+        return LINKER.downcallHandle(WTSAPI32.lookup(methodName).orElseThrow(), fd);
+    }
+
+    private static final MethodHandle WTSEnumerateProcessEx = methodHandle("WTSEnumerateProcessesA", FunctionDescriptor.of(JAVA_BOOLEAN, JAVA_INT, JAVA_INT, JAVA_INT, ADDRESS, ADDRESS));
+
+    private static final MemoryLayout _WTS_PROCESS_INFOA = MemoryLayout.structLayout(
+        JAVA_INT.withName("SessionId"),
+        JAVA_INT.withName("ProcessId"),
+        JAVA_CHAR.withName("pProcessName"),
+        ADDRESS.withName("pUserSid")
+    ).withName("WTS_PROCESS_INFOA");
+
+    /**
+     * Refer to <a href="https://github.com/VFPX/Win32API/blob/master/libraries/wtsapi32/WTSEnumerateProcesses.md">Win32API</a>
+     */
+    public static void enumerateProcesses() {
```

Comment:
I'm thinking maybe we can have `enumerateProcesses` and all these methods in these classes to defer the allocation to the caller?

I.e. `try (Memory session = Memory.openConfined())`  will be left to the caller to try-with and just have these classes as utils

Also looked at jextract looks funky 👀